src/main/java/socialite/storage/JsonCommandHistoryStorage.java

```diff
@@ -0,0 +1,60 @@
+package socialite.storage;
+
+import static java.util.Objects.requireNonNull;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import socialite.commons.exceptions.DataConversionException;
+import socialite.commons.util.FileUtil;
+import socialite.commons.util.JsonUtil;
+import socialite.model.ReadOnlyCommandHistory;
+
+public class JsonCommandHistoryStorage implements CommandHistoryStorage {
```

Comment:
Might be good to include a JavaDoc comment at the top to give an overview of this class/file.